app/app/static/custom/ourscript.js

```diff
@@ -342,3 +342,43 @@ window.onload = async (event) => {
 	rerenderTables();
 	
 };
+
+$('#dateSubmit').on('click', function(e) {
+	const startDate = document.getElementById("startDate").value;
+	const endDate = document.getElementById("endDate").value;
+	let startTime = startDate + " 00:00:00.0"
+	let endTime = endDate + " 23:59:59.9"
+
+	requestCSV({
+		startTime,
+		endTime
+	})
+})
+
+const requestCSV = async (data) => {
+	// This function requests a csv from the application with the data
```

Comment:
You might want to specify what the object data needs to contain

Reply:
I'm not sure how to go about doing that

Reply:
Just a comment what the variable object should contain

Reply:
Sure thing